api/alembic/versions/baa3e0182740_add_hfi_ready_table.py

```diff
@@ -0,0 +1,53 @@
+"""Add hfi_ready table
+
+Revision ID: baa3e0182740
+Revises: ac65354014bd
+Create Date: 2022-05-17 16:20:14.867265
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'baa3e0182740'
+down_revision = 'ac65354014bd'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic ###
+    op.create_table('hfi_ready',
+                    sa.Column('id', postgresql.UUID(as_uuid=True), nullable=False),
+                    sa.Column('hfi_request_id', sa.Integer(), nullable=False),
+                    sa.Column('planning_area_id', sa.Integer(), nullable=False),
+                    sa.Column('ready', sa.Boolean(), nullable=False),
+                    sa.Column('create_timestamp', sa.TIMESTAMP(timezone=True), nullable=False),
+                    sa.Column('create_user', sa.String(), nullable=False),
+                    sa.Column('update_timestamp', sa.TIMESTAMP(timezone=True), nullable=False),
+                    sa.Column('update_user', sa.String(), nullable=False),
+                    sa.ForeignKeyConstraint(['hfi_request_id'], ['hfi_request.id'], ),
+                    sa.ForeignKeyConstraint(['planning_area_id'], ['planning_areas.id'], ),
+                    sa.PrimaryKeyConstraint('id'),
+                    comment='Marks whether a planning area is ready for a particular HFI Request'
+                    )
+    op.create_index(op.f('ix_hfi_ready_create_timestamp'), 'hfi_ready', ['create_timestamp'], unique=False)
+    op.create_index(op.f('ix_hfi_ready_hfi_request_id'), 'hfi_ready', ['hfi_request_id'], unique=False)
+    op.create_index(op.f('ix_hfi_ready_planning_area_id'), 'hfi_ready', ['planning_area_id'], unique=False)
+    op.create_index(op.f('ix_hfi_ready_ready'), 'hfi_ready', ['ready'], unique=False)
+    op.create_index(op.f('ix_hfi_ready_update_timestamp'), 'hfi_ready', ['update_timestamp'], unique=False)
+    op.drop_index('ix_planning_weather_stations_id', table_name='planning_weather_stations')
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic ###
+    op.create_index('ix_planning_weather_stations_id', 'planning_weather_stations', ['id'], unique=False)
```

Comment:
There's index flip flopping happening here - please look into it.

Reply:
It's because of the change discussed here: https://github.com/bcgov/wps/pull/1973#discussion_r864334139

It removed the `index=True` parameter since we already get a primary key index by default:
![Screen Shot 2022-05-18 at 10 23 34 AM](https://user-images.githubusercontent.com/1447136/169104316-21b6eab3-074b-49a7-ab9e-9c484a4a89f6.png)

I must have manually deleted the migration from the PR that made the change.